packages/manager/apps/dedicated/client/app/dedicated/server/ipmi/dedicated-server-ipmi.controller.js

```diff
@@ -549,13 +549,18 @@ angular
 
       // ------------Start KVM URL------------
       $scope.getIpmiKvmUrl = function getIpmiKvmUrl() {
+        const withGeolocation =
+          !includes(['HIL_1', 'VIN_1'], $scope.server.datacenter) &&
+          coreConfig.isRegion('US');
```

Comment:
Maybe the `coreConfig.isRegion` could be replaced with feature flipping

Reply:
Thanks Marie, goal here is to pass `withGeolocation` parameter (true/false) when we start an IPMI session not to activate/deactivated a feature.

Reply:
Adding geolocation could be seen as a feature in some sense and here it seems that it's a feature only available for US region so it could make sense to use feature flipping imo (we could want to add geolocation in some future for other regions let's say)

Reply:
The KVM console don't accept a connection without a trusted IP address.
So we have to pass `withGeolocation: true` to pass the IP addess.